joi-validations/topics.validations.js

```diff
@@ -5,6 +5,16 @@ const TopicValidation = {
     query: {
       name: string.required()
     }
+  },
+  checkName: {
+    query: {
+      name: string.required()
+    }
+  },
+  create: {
+    body: {
+      name: string.required()
```

Comment:
@brm-stnd @ibramastana please confirm with product team, this should have minimum query length. We don't want to search the DB with name less than 3 since that will be heavy.